src/Npgsql/NpgsqlSpanStream.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Npgsql
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public abstract class NpgsqlSpanStream : NpgsqlStream
```

Comment:
This one isn't exposed to public API, so let's make it internal and remove meaningless comments.

Reply:
This one must remain public as it is used as base implementation by public `NpgsqlRawCopyStream` class.
I have added a more explicit comment ^^